types/stripe/index.d.ts

```diff
@@ -5,11 +5,12 @@
 //                 Amrit Kahlon <https://github.com/amritk/>
 //                 Adam Cmiel <https://github.com/adamcmiel>
 //                 Justin Leider <https://github.com/jleider>
+//                 Kamil Gałuszka <https://github.com/galuszkak>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
```

Comment:
So what I understand, export is legit if you have real JavaScript in node_modules. But here we are using stripe service that always be global from 3rd party service. 

Angular is something that you add and install in Your node_modules, but with Stripe or Google Maps you are just adding to html `<script src="https://js.stripe.com/v3/"></script>` So there is a global available through the project. 

So I think `export` is here invalid as this isn't a definition file to anything that is in npm registry.  Does that make sense for You @amritk ?